src/fsAsync.ts

```diff
@@ -16,22 +16,24 @@ export interface ReaddirRecursiveFile {
 }
 
 async function readdirRecursiveHelper(options: {
```

Comment:
This seems broken to me at the moment - you've changed the signsture of readdirRecursiveHelper but not changed any of its call sites.